rebar.config.script

```diff
@@ -15,6 +15,18 @@
 ConfigureEnv = filename:join(filename:dirname(SCRIPT), "config.erl"),
 os:putenv("COUCHDB_CONFIG", ConfigureEnv).
 
+OTP_VSN = erlang:system_info(compat_rel),
+
+Features = lists:filter(fun({_K, Flag}) -> Flag end, [
+    {'HAS_CALLBACKS', OTP_VSN > 14}
+]),
+
+Properties = Features ++ [
+    {'OTP_VSN', OTP_VSN},
+    {'OTP_RELEASE', erlang:system_info(otp_release)}
+],
```

Comment:
I guess @iilyak tried to have more generic solution, but yours simpler and clearer to read. Nice!